microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -48,90 +47,94 @@ See also:
 * [Analyze selector performance during Recalculate Style events](../../../evaluate-performance/selector-stats.md)
 * [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)
 
+
 <!-- ====================================================================== -->
-## The Issues tool warns you when CSS properties trigger reflow 
+## The Issues tool warns you when CSS properties trigger reflow
+<!-- todo: is this a change to the Issues tool, Styles pane, or both? -->
 
-<!-- Subtitle: Reflow is when the browser re-calculates the positions and geometries of elements in the page and is user-blocking so limit reflow as much as possible for better performance.-->
+<!-- Subtitle: The Styles pane shows a wavy underline on CSS properties that trigger reflow, to help you limit reflow and avoid blocking user interaction.  Reflow is when the browser re-calculates the positions and geometries of elements in the page. -->
 
 <!-- Reviewer: Vidal Guillermo Diazleal Ortega -->
 
-Reflow is the name of the web browser process for re-calculating the positions and geometries of elements in the document, for the purpose of re-rendering part or all of the document. Because reflow is a user-blocking operation in the browser, from a performance perspective, we recommend limiting reflow as much as possible to keep your web content smooth and interactable. Some CSS properties don't trigger reflow because they run on the compositor thread in the browser, like [transform](https://developer.mozilla.org/docs/Web/CSS/transform) and [opacity](https://developer.mozilla.org/docs/Web/CSS/opacity).
+Reflow is the web browser process for recalculating the positions and geometries of elements in the document, to re-render part or all of the document.  Because reflow is a user-blocking operation in the browser, for performance, we recommend limiting reflow as much as possible, to keep your web content smooth and responsive to interaction.
 
-Now, the **Styles** pane in the **Elements** tool adds a wavy underline for CSS properties that do trigger reflow.  Hovering over the wavy underline will provide a tooltip that shows the issue
+Some CSS properties don't trigger reflow, because they run on the compositor thread in the browser - for example, the [transform](https://developer.mozilla.org/docs/Web/CSS/transform) and [opacity](https://developer.mozilla.org/docs/Web/CSS/opacity) properties.
+
+Now, the **Styles** pane in the **Elements** tool adds a wavy underline on CSS properties that trigger reflow.  To view the issue in a tooltip, hover over the wavy underline:
 
```

Comment:
reword tooltip from:
(Shift + Click) or (Right Click + 'View issues') to view recommended fix in Issues panel.
to:
(Shift+click) or (right-click > 'View issues') to view recommended fix in Issues panel.

screenshot showing the tooltip:
https://github.com/mikehoffms/edge-developer/raw/user/mikehoffms/whats-new-114/microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114-images/css-reflow-issue-styles-pane.png
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui -
Select **Shift+Enter**.
right-click the

Reply:
dev is logging bug